tests/CoreFX/CoreFX.issues.json

```diff
@@ -144,6 +144,66 @@
                 {
                     "name": "BasicEventSourceTests.TestsUserErrors.Test_BadEventSource_MismatchedIds",
                     "reason": "Assert.Equal() Failure Expected: 1 Actual: 2"
+                },
```

Comment:
Note: This is added here because we've now added System.Runtime in the runtime, but these CoreFX tests don't know that this exists. I've already modified the CoreFX test to reflect this change in https://github.com/dotnet/corefx/pull/35227, but this change is NOT reflected in CoreCLR's CoreFX CI leg because the version of CoreFX test blob we download for running this is outdated (dates back to ~8/2018). Once that issue is fixed I'll enable these tests again.